accesscontroltool-package/src/main/META-INF/vault/filter.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <workspaceFilter version="1.0">
-	<filter root="/apps/netcentric/actool"/>       
+	<filter root="/apps/netcentric/actool"/>  
+	<filter root="/home/users/system/netcentric/actool/acl/ynUbs6nnylcpRBs0eXdv"/>

```

Comment:
I would use the path /home/users/system/actool/actool-service (if you do not use the api to create a user but a package, you can use an unobfuscated path)